conf/test.config

```diff
@@ -14,10 +14,10 @@ params {
   // Limit resources so that this can run CI
   max_cpus   = 2
   max_memory = 6.GB
-  max_time   = 48.h
+  max_time   = 6.h
 
   // Input data
-  input = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/samplesheet.csv'
+  input = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/rnaseq_v3.1/samplesheet_test.csv'
```

Comment:
I would have gone for a simpler url such as `https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/3.1/test.csv`, but not important

Reply:
Agree! Fixed https://github.com/nf-core/test-datasets/pull/247 and https://github.com/nf-core/rnaseq/pull/617/commits/94a98d4555b9b7ae01131d601e2e2af0869943c0

Reply:
Agree! Fixed https://github.com/nf-core/test-datasets/pull/247 and https://github.com/nf-core/rnaseq/pull/617/commits/94a98d4555b9b7ae01131d601e2e2af0869943c0